### PR TITLE
[2.7] bpo-30283: Add NEWS entry for backported regrtest

### PR DESCRIPTION
**Pull request by vstinner**

### CODE REVIEW

**Comment by vstinner** [comment]
@brettcannon, @larryhastings: Do you know how to test how this NEWS entry with newlines will be formatted in Python 2.7? How do I regenerate/update Misc/NEWS?

This is my first PR using blurb :-)

File: Misc/NEWS.d/next/Tests/2017-06-26-11-24-14.bpo-30283.qCQmlm.rst [hunk 1]
```diff
@@ -0,0 +1,29 @@
+regrtest: Enhance regrtest and backport features from the master branch:
+
+* Add --testdir option
```
Comment (serhiy-storchaka) [inline Misc/NEWS.d/next/Tests/2017-06-26-11-24-14.bpo-30283.qCQmlm.rst]: If all items are full sentences, they should end with a period.

Comment (vstinner) [reply Misc/NEWS.d/next/Tests/2017-06-26-11-24-14.bpo-30283.qCQmlm.rst]: done

**Comment by brettcannon** [comment]
@haypo I think `blurb merge` with the output directed to some temp file will do what you want. I just opened python/core-workflow#147 to at least test the reST formatting is good.

**Comment by vstinner** [comment]
> @haypo I think blurb merge with the output directed to some temp file will do what you want.

Oh, I wasn't aware of the "blurb merge" command. I tried "blurb --help" but the wizzard started :-)

**Pull request merged**